apollos-church-api/package.json

```diff
@@ -5,6 +5,8 @@
   "main": "lib/index.js",
   "scripts": {
     "setup": "./scripts/init.sh",
+    "migrator": "node ./lib/migrator.js",
+    "migrator:production": "heroku --app YOUR_HEROKU_APP_NAME_HERE run yarn migrator up",
```

Comment:
Could we make migrations automatic on the Heroku app? Is that not the purpose of the `AUTO_MIGRATE` variable?  I just hate introducing named specific things to the template that don't work out of the box and require more documentation and setup.

Reply:
I don't think migrations should be automatic ever. Except for "power users" who want to run migrations automatically to make development easier. I'm fine deleting the coded that let's migrations run automatically.

Reply:
what about just leaving this line off then? I like the shortcut but I don't like the specific name and setup required even more. I feel like someone should have to ask and know what they're doing before running migrations in production. having this line in the setup readme should suffice.

Reply:
fine by me ;) I had assumed that if I left this line out you were going to ask me to add it in

Reply:
```suggestion
```